src/test/java/com/meilisearch/integration/TasksTest.java

```diff
@@ -202,6 +205,38 @@ public void testClientGetTasksAllQueryParameters() throws Exception {
         assertThat(result.getNext(), is(notNullValue()));
         assertThat(result.getResults().length, is(notNullValue()));
     }
+    /** Test to check if task list is reversed when enabled */
+    @Test
+    public void testGetTasksInReverse() {
+        String indexUid = "tasksOnReverseOrder";
+        Date currentTime = Date.from(Instant.now());
+        TestData<Movie> testData = this.getTestData(MOVIES_INDEX, Movie.class);
+
+        TaskInfo taskA = client.index(indexUid).addDocuments(testData.getRaw());
+        TaskInfo taskB = client.index(indexUid).addDocuments(testData.getRaw());
+
+        client.waitForTask(taskA.getTaskUid());
+        client.waitForTask(taskB.getTaskUid());
+
+        Task[] defaultTaskList =
+                client.getTasks(new TasksQuery().setAfterEnqueuedAt(currentTime)).getResults();
+        Task[] reversedTaskList =
+                client.getTasks(new TasksQuery().setAfterEnqueuedAt(currentTime).setReverse(true))
+                        .getResults();
```

Comment:
Can you map only the task ids here, and then you assert them later, so the test will be less complex to read.